lib/console.js

```diff
@@ -21,7 +21,7 @@ function Console(stdout, stderr) {
   Object.defineProperty(this, '_stdout', prop);
   prop.value = stderr;
   Object.defineProperty(this, '_stderr', prop);
-  prop.value = {};
```

Comment:
@caitp I literally just changed from `Object.create(null)` to a `Map`. For future reference, if there ends up being a problem, it will be a very simple fix. I think I like @rvagg's idea to experiment with a `Map` here since performance is not crucial.